libs/sdk-model/src/execution/filter/index.ts

```diff
@@ -246,6 +246,18 @@ export type IFilter =
     | IMeasureValueFilter
     | IRankingFilter;
 
+/**
+ * Tiger specific wrapper for IFilter, adding 'applyOnResult' property influencing the place of filter application.
+ * This property could very well be part of each I*Filter but since it's needed only internally so far it's better
+ * to not change the public API.
+ *
+ * @internal
+ */
+export interface IFilterWithApplyOnResult {
+    filter: IFilter;
+    applyOnResult: boolean | undefined;
+}
+
```

Comment:
have you considered moving this to sdk-backend-tiger? as is defined now, it is only really useful in that package and I'm suspecting that when we make the applyOnResult truly public, we will include it in I*Filter.

I mean.. even the comment says that + mentions it is only used internally

Reply:
Yes, I've considered that I eventually thought that it's better to include it here, mainly for the purpose of visibility so that once someone decides to make this public he knows where to start. And even if we made it public but still tiger only (regarding the functionality) it would still belong here, right?

So unless you have a strong objection I would keep it as is.

Reply:
if we make it public, then it will be part of IFilter and won't be made public in this form. as it is now, even if we wanted to make it public for tiger in this form, you would not be able to pass it to any of the React components because they expect IFilter.